MAUI/MauiAppWithBroker/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,84 @@
+{
+  "Sample": {
+    "Title": "MAUI using MSAL.NET and a broker to acquire an access token and call Microsoft Graph API",
+    "Level": 200,
+    "Client": "MAUI (iOS, Android, UWP)",
+    "RepositoryUrl": "active-directory-xamarin-native-v2",
+    "Endpoint": "AAD v2.0",
+    "Library": "Microsoft Graph"
+  },
```

Comment:
active-directory-xamarin-native-v2

Reply:
I think we'd need to make a new repository if we wanted to change that. No?